                    0  1            2   ...                 19     20         21
0    2023-05-28, 17:55        Žalgiris  ...  2021-11-28, 15:00  3 : 2    Džiugas
1    2023-05-28, 16:00     K. Žalgiris  ...                NaN    NaN        NaN
2    2023-05-27, 19:00        Riteriai  ...                NaN    NaN        NaN
3    2023-05-27, 17:00        Šiauliai  ...                NaN    NaN        NaN
4    2023-05-26, 18:00          Sūduva  ...                NaN    NaN        NaN
..                 ... ..          ...  ...                ...    ...        ...
739                NaN          Sūduva  ...  2021-03-12, 18:00  2 : 0  Hegelmann
740                NaN          Sūduva  ...  2021-03-07, 15:00  0 : 1  Panevėžys
741                NaN         Dainava  ...  2021-03-06, 15:00  0 : 2    Nevėžis
742                NaN         Džiugas  ...  2021-03-06, 13:00  2 : 1  Hegelmann
743                NaN     K. Žalgiris  ...  2021-03-05, 18:00  1 : 0      Banga
[744 rows x 22 columns]
Traceback (most recent call last):
  File "/home/orestas41/project-FootballPredict/lithuanian-football-league-predictor/pre-processing/run.py", line 185, in <module>
    go(args)
  File "/home/orestas41/project-FootballPredict/lithuanian-football-league-predictor/pre-processing/run.py", line 59, in go
    df.columns = ["Date", "Blank", "Home", "Result", "Away", "Location"]
  File "/home/orestas41/miniconda3/envs/mlflow-fe275cdffdb91d423bbbc9efcaa0b6a9fba88e1b/lib/python3.10/site-packages/pandas/core/generic.py", line 5596, in __setattr__
    return object.__setattr__(self, name, value)
  File "pandas/_libs/properties.pyx", line 70, in pandas._libs.properties.AxisProperty.__set__
  File "/home/orestas41/miniconda3/envs/mlflow-fe275cdffdb91d423bbbc9efcaa0b6a9fba88e1b/lib/python3.10/site-packages/pandas/core/generic.py", line 769, in _set_axis
    self._mgr.set_axis(axis, labels)
  File "/home/orestas41/miniconda3/envs/mlflow-fe275cdffdb91d423bbbc9efcaa0b6a9fba88e1b/lib/python3.10/site-packages/pandas/core/internals/managers.py", line 214, in set_axis
    self._validate_set_axis(axis, new_labels)
  File "/home/orestas41/miniconda3/envs/mlflow-fe275cdffdb91d423bbbc9efcaa0b6a9fba88e1b/lib/python3.10/site-packages/pandas/core/internals/base.py", line 69, in _validate_set_axis
    raise ValueError(
ValueError: Length mismatch: Expected axis has 22 elements, new values have 6 elements